### PR TITLE
Update migratingtomlapi.md

### PR DESCRIPTION
**Pull request by Briancoughlin**
**Select the type of change:**

- [x] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
MInor updated to example and content -->

**Issue Number:** 
#106 #105

### CODE REVIEW

**Review by lkrell — APPROVED** [review]
Looks good to me. Luke or Fatih can definitely 👍  👎

File: docs/migration/migratingtomlapi.md [hunk 1]
```diff
@@ -147,16 +147,16 @@ public class MyMLAPIExample : NetworkBehaviour
     public NetworkVariable<float> MyNetworkVariable = new NetworkVariable<float>();
     public override void NetworkStart()
     {
-        InvokeClientRpcOnEveryone(ClientRpcExample, 10f);
-        InvokeServerRpc(ServerRpcExample, 10f);
+        InvokeClientRpcOnEveryone(ExampleClientRpc, 10f);
+        InvokeServerRpc(ExampleServerRpc, 10f);
     }
     [ServerRPC]
```
Comment (MFatihMAR) [inline docs/migration/migratingtomlapi.md:153]:
```suggestion
    [ServerRpc]
```

File: docs/migration/migratingtomlapi.md [hunk 2]
```diff
@@ -147,16 +147,16 @@ public class MyMLAPIExample : NetworkBehaviour
...
-    public void ServerRpcExample(float x)
+    public void ExampleServerRpc(float x)
     {
         Debug.Log(“Runs on server”);
     }
     [ClientRPC]
```
Comment (MFatihMAR) [inline docs/migration/migratingtomlapi.md:158]:
```suggestion
    [ClientRpc]
```

File: docs/migration/migratingtomlapi.md [hunk 3]
```diff
@@ -147,16 +147,16 @@ public class MyMLAPIExample : NetworkBehaviour
     public NetworkVariable<float> MyNetworkVariable = new NetworkVariable<float>();
     public override void NetworkStart()
     {
-        InvokeClientRpcOnEveryone(ClientRpcExample, 10f);
-        InvokeServerRpc(ServerRpcExample, 10f);
+        InvokeClientRpcOnEveryone(ExampleClientRpc, 10f);
+        InvokeServerRpc(ExampleServerRpc, 10f);
```
Comment (MFatihMAR) [inline docs/migration/migratingtomlapi.md:151]:
```suggestion
        ExampleClientRpc(10f);
        ExampleServerRpc(10f);
```

**Comment by MFatihMAR** [comment]
LGTM! 🚀

**Review by MFatihMAR — APPROVED** [review]

**Pull request merged**